modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js

```diff
@@ -204,61 +202,44 @@ const LocalizableText = ({
 		? currentInternalValue
 		: predefinedValue;
 
-	const {portletNamespace} = usePage();
-
 	useEffect(() => {
-		const onClickDDMFormSettingsButton = function () {
-			const translationManager = Liferay.component('translationManager');
-
-			const newAvailableLocales = translationManager.get(
-				'availableLocales'
-			);
-
-			const {availableLocales} = {
-				...transformAvailableLocales(
-					[...newAvailableLocales],
-					defaultLocale,
-					currentValue
-				),
-			};
-
-			const newEditingLocale = transformEditingLocale({
-				defaultLocale,
-				editingLocale: newAvailableLocales.get(
-					translationManager.get('editingLocale')
-				),
-				value: currentValue,
-			});
+		const translationManager = Liferay.component('translationManager');
 
-			setCurrentAvailableLocales(availableLocales);
+		if (!translationManager) {
+			return;
+		}
 
-			setCurrentEditingLocale(newEditingLocale);
+		const newAvailableLocales = translationManager.get('availableLocales');
 
-			setCurrentInternalValue(
-				getEditingValue({
-					defaultLocale,
-					editingLocale: newEditingLocale,
-					fieldName,
-					value: currentValue,
-				})
-			);
+		const {availableLocales} = {
+			...transformAvailableLocales(
+				[...newAvailableLocales],
+				defaultLocale,
+				currentValue
+			),
 		};
 
-		const clickDDMFormSettingsButton = delegate(
-			document.body,
-			'click',
-			`#${portletNamespace}ddmFormInstanceSettingsIcon`,
-			onClickDDMFormSettingsButton
-		);
+		const newEditingLocale = transformEditingLocale({
+			defaultLocale,
+			editingLocale: newAvailableLocales.get(
+				translationManager.get('editingLocale')
+			),
+			value: currentValue,
+		});
 
-		return () => clickDDMFormSettingsButton.dispose();
-	}, [
-		currentAvailableLocales,
-		currentValue,
-		defaultLocale,
-		fieldName,
-		portletNamespace,
-	]);
+		setCurrentAvailableLocales(availableLocales);
+
+		setCurrentEditingLocale(newEditingLocale);
+
+		setCurrentInternalValue(
```

Comment:
@miltonmc, I would like to keep it the way it is so I don't spend more time on this bug because these **setState** already exist in **LocalizableText** and I don't see a problem with keeping it until I have time to refactor it

Reply:
Ok, no problem!